Tutorials/whatiswot/10-Application_Domains/10-Application-Domains.txt

```diff
@@ -0,0 +1,49 @@
+In this video, we will give a introduction to the W3C Web of Things including its application domains and we will continue with architecture patterns, and building blocks in the upcoming videos. 
+
+So let's start with what is WoT. 
+
+It is a set of standards  defined by the World Wide Web Consortium (W3C)  for the interoperability of  different Internet of Things platforms and application domains. 
+
+The WoT aims to preserve and complement existing IoT standards and solutions  by providing a standardized approach for connecting and integrating   smart devices, services, and applications into the Web.
```

Comment:
The animations here feel out of sync with the text. The sentence starts by mentioning the W3C but the animation shows a document structure and only after a **few seconds** shows the consortium animation. It is like the video can't keep up with what is narrated. Maybe We should choose one of the two animations?

Reply:
Let's do such that we move the W3C mention to the end of the sentence. If it is still too fast, we can remove the W3C slide/animation.